jest.unit.conf.json

```diff
@@ -50,7 +50,8 @@
     "/frontend/src/metabase/internal/",
     "/frontend/test/",
     ".styled.{jsx,tsx}",
-    ".info.js"
+    ".info.js",
+    "**/*.unit.*"
```

Comment:
@nemanjaglumac, I know this is out of the scope of this PR, but WDYT about excluding [Storybook files from code coverage](https://github.com/metabase/metabase/blob/714e8cdcd4c22aa89a748e38e6ec4591c5dfc5d6/jest.unit.conf.json#L44)? Just noticed it's there while reviewing the PR

Reply:
Well, this is extremely weird.
@ariya wanted to exclude stories in this PR https://github.com/metabase/metabase/pull/22661, and you approved it.
But that's where stories were actually added to the `collectCoverageFrom`.

I guess it was just a mistake that slipped, but have to confirm first.

Reply:
Yeah, looks like a double accident to me :( Thanks for fixing this! 🙏